node/src/neighborhood/mod.rs

```diff
@@ -106,6 +108,9 @@ pub struct Neighborhood {
     db_password_opt: Option<String>,
     logger: Logger,
     tools: NeighborhoodTools,
+    exit_locations_opt: Option<HashMap<usize, Vec<String>>>,
+    exit_countries: Vec<String>, //if we cross number of countries used in one workflow, we want to change this member to HashSet<String>
+    fallback_routing: bool,
```

Comment:
I may also come back and change _this_ comment, but right now I think that these two fields could possibly be replaced with a three-valued enum: no exit-location preference, exit-location preference with fallback, and exit-location preference without fallback.